model/pixel_to_millimeter.h

```diff
@@ -12,6 +12,7 @@
 #define DXTBX_MODEL_PIXEL_TO_MILLIMETER_H
 
 #include <scitbx/vec2.h>
+#include <scitbx/vec3.h>
```

Comment:
Don't think this is required?

Reply:
Not sure why this was added. Thanks I will delete.